APItest.xs:1087:16: warning: format ‘%d’ expects argument of type ‘int’, but argument 2 has type ‘line_t {aka long unsigned int}’

Compiling 32bit Perl from current git blead branch cause following compile warning:
```
gcc -m32 -c   -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2 -Wno-deprecated-declarations -Wall -Werror=declaration-after-statement -Werror=pointer-arith -Wextra -Wc++-compat -Wwrite-strings -O2   -DVERSION=\"1.04\" -DXS_VERSION=\"1.04\" -fPIC "-I../.."   APItest.c
In file included from ../../perl.h:5543:0,
                 from APItest.xs:10:
APItest.xs: In function ‘THX_parse_keyword_subsignature’:
APItest.xs:1087:16: warning: format ‘%d’ expects argument of type ‘int’, but argument 2 has type ‘line_t {aka long unsigned int}’ [-Wformat=]
       newSVpvf("nextstate:%d", cCOPx(kid)->cop_line)));
                ^
../../embed.h:400:61: note: in definition of macro ‘op_append_list’
 #define op_append_list(a,b,c) Perl_op_append_list(aTHX_ a,b,c)
                                                             ^
APItest.xs:1085:42: note: in expansion of macro ‘newSVOP’
   retop = op_append_list(OP_LIST, retop, newSVOP(OP_CONST, 0,
                                          ^
```
Specifying incorrect type in variadic printf-like function may cause undefined behavior.
Type `line_t` is defined as `typedef U32 line_t;` in handy.h

> `./Configure -de -Dusedevel -Dcc='gcc -m32' -Dlibs='-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc'`

I am attempting to reproduce your report.  When I configure with:
```
./Configure -de -Dusedevel -Dcc='gcc -m32' -Dlibs='-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc'
```
on this platform:
```
$ uname -mrs
Linux 4.15.0-70-generic x86_64
$ gcc --version | head -1
gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
```
... `Configure` quickly bombs out:
```
Good, your tr supports [:lower:] and [:upper:] to convert case.
Using [:upper:] and [:lower:] to convert case.
/usr/bin/ld: cannot find Scrt1.o: No such file or directory
/usr/bin/ld: cannot find crti.o: No such file or directory
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/7/libgcc.a when searching for -lgcc
/usr/bin/ld: cannot find -lgcc
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/7/libgcc.a when searching for -lgcc
/usr/bin/ld: cannot find -lgcc
collect2: error: ld returned 1 exit status
You need to find a working C compiler.
Either (purchase and) install the C compiler supplied by your OS vendor,
or for a free C compiler try http://gcc.gnu.org/
I cannot continue any further, aborting.
```
What am I missing?

Thank you very much.
Jim Keenan

--- a/ext/XS-APItest/APItest.pm
+++ b/ext/XS-APItest/APItest.pm
@@ -5,7 +5,7 @@ use strict;
 use warnings;
 use Carp;
 
-our $VERSION = '1.04';
+our $VERSION = '1.05';
 
 require XSLoader;
 

--- a/ext/XS-APItest/APItest.xs
+++ b/ext/XS-APItest/APItest.xs
@@ -1084,7 +1084,7 @@ static OP *THX_parse_keyword_subsignature(pTHX)
 		seen_nextstate++;
 		retop = op_append_list(OP_LIST, retop, newSVOP(OP_CONST, 0,
 		    /* newSVpvf("nextstate:%s:%d", CopFILE(cCOPx(kid)), cCOPx(kid)->cop_line))); */
-		    newSVpvf("nextstate:%d", cCOPx(kid)->cop_line)));
+		    newSVpvf("nextstate:%u", (unsigned int)cCOPx(kid)->cop_line)));
 		break;
 	    case OP_ARGCHECK: {
 		UNOP_AUX_item *aux = cUNOP_AUXx(kid)->op_aux;

